source/_components/sensor.irish_rail_transport.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "Irish Rail Transport"
+description: "Instructions how to integrate timetable data for travelling on Irish Rail within Home Assistant."
+date: 2017-10-15 16:50
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: irishrail.png
+ha_category: Transport
+ha_iot_class: "Cloud Polling"
+ha_release: 0.56
+---
+
+
+The `irish_rail_transport` sensor will give you the time until the next two departures (within 90 minutes) from an Irish Rail station using the RTPI information.
+
+A station name is the full station name as specified on the Irish Rail search site, for example `Tara Street` or `Dublin Connolly`.
+
+To activate the sensor add the data to your `configuration.yaml` file as shown in the example:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: irish_rail_transport
+    station: "Tara Street"
+    direction: Southbound
+    destination: Greystones
+    stops_at: "Dun Laoghaire"
+    name: "To Greystones"
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.